Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
-        precisemaneuver_keybindings_pageconics          = Change orbit mode (w/Alt)
-        precisemaneuver_keybindings_encfocus            = Focus on next encounter
-        precisemaneuver_keybindings_patchesmore         = Show more patches
-        precisemaneuver_keybindings_patchesless         = Show fewer patches
-        precisemaneuver_keybindings_timex10             = Toggle x10 time
-        precisemaneuver_keybindings_nextnode            = Next maneuver node
-        precisemaneuver_keybindings_prevnode            = Prev maneuver node
-        precisemaneuver_keybindings_delnode             = Delete maneuver node
-                
-        precisemaneuver_date_format                     = Year <<1>>, Day <<2>>, <<3>>:<<4>>:<<5>>
-        precisemaneuver_date_format_compact             = Y<<1>> D<<2>> <<3>>:<<4>>:<<5>>
-                
-        precisemaneuver_node                            = Node
+   es-es {
+        precisemaneuver_caption                         = MANIOBRA PRECISA
+        precisemaneuver_keybindings_caption             = ATAJOS DE TECLADO MANIOBRA PRECISA
+                
+        precisemaneuver_mainmenu_enable                 = Habilitar
+        precisemaneuver_mainmenu_keybindings            = Establecer atajos
+        precisemaneuver_mainmenu_background             = Fondo
+        precisemaneuver_mainmenu_scale                  = Scala de la GUI
+        precisemaneuver_mainmenu_components             = COMPONENTES
+                
+        precisemaneuver_module_pager                    = Buscador de Maniobras
+        precisemaneuver_module_saver                    = Plantillas de Maniobras
+        precisemaneuver_module_precise_input            = Entrada precisa
+        precisemaneuver_module_orbit_tools              = Herramientas de Órbita
+        precisemaneuver_module_gizmo                    = Gadget de Maniobra
+        precisemaneuver_module_next_encounter           = Siguiente encuentro
+        precisemaneuver_module_ejection                 = Ángulo de eyeción
+        precisemaneuver_module_info                     = Informacion orbital
+        precisemaneuver_module_patches                  = Control de parches
+                
+        precisemaneuver_keybindings_set                 = Activar
+        precisemaneuver_keybindings_unset               = Desactivar
+                
+        precisemaneuver_keybindings_enable              = Mostrar/ocultar ventana
+        precisemaneuver_keybindings_proginc             = Incremenar progrado
+        precisemaneuver_keybindings_progdec             = Reducir progrado
+        precisemaneuver_keybindings_progzero            = Anular progrado
+        precisemaneuver_keybindings_norminc             = Incrementar normal
+        precisemaneuver_keybindings_normdec             = Reducir normal
+        precisemaneuver_keybindings_normzero            = Anular normal
```

Comment:
i'm not sure here whats mean "zero normal" in the UI, probably here should be "Cero normal"